src/styles/styles/_card.scss

```diff
@@ -0,0 +1,118 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  height: 408px;
```

Comment:
don't set height for card. It should be set by height of elements inside

Reply:
Not fixed. Card's height shouldn't be hardcored